src/DatabasesFileSystem.ts

```diff
@@ -84,7 +87,15 @@ export class DatabasesFileSystem extends AzExtTreeFileSystem<IEditableTreeItem>
     }
 
     protected async findItem(context: IActionContext, query: AzExtItemQuery): Promise<IEditableTreeItem | undefined> {
-        const node: IEditableTreeItem | undefined = await super.findItem(context, query);
+        let node: IEditableTreeItem | undefined = await super.findItem(context, query);
+        if (!node && ext.connectedMongoDB) {
```

Comment:
Hmm this `if` case does nothing to confirm the query is actually for a collection tree item. Can we fix that? One idea I had:
```
if (!node) {
   const parentId: string = // get parentId from query.id
   const parentNode: IEditableTreeItem | undefined = ext.tree.findTreeItem(parentId);
   if (parentNode instanceof MongoDatabaseTreeItem) {
        // create mongo collection tree item
   }
}
```

Also feels a bit more robust than using `ext.connectedMongoDB`